src/microbe_stage/Membrane.cs

```diff
@@ -101,7 +102,14 @@ public float WigglyNess
     {
         get
         {
-            return wigglyNess;
+            if (cellDimensions > 1)
+            {
+                return wigglyNess / (cellDimensions * sizeWigglyNessDampeningFactor);
```

Comment:
Isn't cellDimensions a constant? Shouldn't this use something like `EncompassingCircleRadius` and also, I think this math should be applied just before passing it to the shader.
One more thing that could be investigated is that if it would be possible to make the shader apply the dampening by detecting how big the vertices are. If that is possible, as I'm not sure if that is possible.

Reply:
Cell dimensions grow inside the InitializeMeshFunction. (Line 399 - 410).
But I can definitely try the EncompassingCircleRadius instead.

Reply:
I didn't realize that cell dimensions changes like that. If it does, then it's fine to use, but I'd still like to see this math moved to where the shader parameter is actually set.

Reply:
Ok. I moved it :)